website/src/lib/users.ts

```diff
@@ -95,7 +95,28 @@ export const getBatchFrontendUserIdFromBackendUser = async (users: { username: s
     const account = externalAccounts.find(
       (a) => a.provider === users[userIdx].auth_method && a.providerAccountId === users[userIdx].username
     );
-    // TODO check why the account is undefined
+    //NOTE: This part gives feedback to understand why user is undefined 
+    // Check if users array is empty
+    if (!users.length) {
+      console.log("Error: users array is empty.");
```

Comment:
Can you maybe add some prefix to all of these console log? just so we know that they came from here

Reply:
Thank you for reviewing, would something like : [getBatchFrontendUserIdFromBackendUser, Undefined user] work ?